pkgs/top-level/all-packages.nix

```diff
@@ -16915,6 +16917,8 @@ with pkgs;
 
   openjump = callPackage ../applications/misc/openjump { };
 
+  open-orienteering-mapper = libsForQt5.callPackage ../applications/gis/open-orienteering-mapper { };
```

Comment:
The application is named OpenOrienteering Mapper so it should probably be named openorienteering-mapper as other distros do: https://repology.org/metapackage/openorienteering-mapper/versions